.gitleaks.toml

```
@@ -20,5 +20,6 @@ paths = [
 
 regexTarget = "match"
 regexes = [
-  '''ABCDEFG1234567890'''
+  '''ABCDEFG1234567890''',
+  '''eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJjZW50cmVvbjY2MjQxIiwiaWF0IjoxNzQ0MDk3MDgxLCJleHAiOjkyMjMzNzIwMzV9.QkrT77i211-CvXoXqaBxRMzxajzA3-DK-DGVrbvJWA8'''
```

Add an other whitelisted secret to keep consistency with develop and added comments of their usage
```suggestion
  '''ABCDEFG1234567890''', # dummy test
  '''eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJjZW50cmVvbjY2MjQxIiwiaWF0IjoxNzQ0MDk3MDgxLCJleHAiOjkyMjMzNzIwMzV9.QkrT77i211-CvXoXqaBxRMzxajzA3-DK-DGVrbvJWA8''', # test base64
  '''ZuFqRKdv0Oz6RItt6elajMugmy1whBrP''' # test agent token
```

---

https://github.com/centreon/centreon-collect/blob/develop/.gitleaks.toml